index.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express');
+const server = express();
+const logger = require('morgan');
+const helmet = require('helmet');
+const PORT = 4000;
+const dbhelper = require('./dbhelper')
+const bcrypt = require('bcryptjs');
+server.use(express.json());
+server.use(helmet());
+server.use(logger('dev'))
+server.disable("etag");
+
+server.post('/api/register', (req, res) => {
+    const newUser = req.body;
```

Comment:
Instead of accessing this objects properties later, it might be helpful to destructure this object into variables that you'll use later.